app/controllers/patients/index.js

```diff
@@ -42,28 +39,12 @@ export default Controller.extend({
 
   populationPatients: computed('selectedPopulations.[]', {
     get() {
-      let selectedPopulations = this.get('selectedPopulations');
-      if (selectedPopulations.length === 0) {
-        return this.get('model.patients');
-      }
-
-      let patients = new A([]);
-      selectedPopulations.forEach(function(population) {
-        population.get('groupList').then((groupList) => {
-          patients.addObjects(groupList.get('patientids'));
-        });
-      });
-
-      return patients;
+      return this.get('model.patients');
     }
   }),
 
-  sortedPatients: computed('selectedPopulations.length', 'filteredPatients.@each.fullName', function sortedPatients() {
-    if (this.get('selectedPopulations.length') === 0) {
-      return this.get('filteredPatients');
-    }
-
-    return this.get('filteredPatients').sortBy('fullName');
+  sortedPatients: computed('filteredPatients.@each.fullName', function sortedPatients() {
```

Comment:
Change this to: 

```js
sortedPatients: computed.reads('filteredPatients')
```

You're using a computed property on a key that you don't do anything with, it's better to use `computed.reads` if you're just making an alias.